multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/flowable/AbortProcessAction.java

```diff
@@ -91,4 +96,8 @@ private Logger getLogger() {
         return LOGGER;
     }
 
+    private static void log(Exception e) {
```

Comment:
Maybe rename to `logDynatraceException` or something similar, because `log` is too generic for class like `AbortProcessAction`

Reply:
Done